src/Generator/AttributeGenerator/AttributeAssemblerFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Code\Generator\AttributeGenerator;
+
+use ReflectionAttribute;
+use ReflectionClass;
+
+final class AttributeAssemblerFactory
```

Comment:
This code can all be inlined into the `AttributeGenerator`: no need to make a separate class out of it.